index.js

```diff
@@ -462,6 +462,28 @@ app.post(
         }
     })
 
+app.post('/smokeTest/setup', async(request, response) => {
+    const {locationid,recipientNumber,twilioNumber} = request.body
+    try{
+        await db.createLocation(locationid, '0', recipientNumber, 17, 15, 150, 30000, 120000, 3000, recipientNumber, twilioNumber, recipientNumber, 9000, locationid, 'door_coreID', 'radar_coreID')
```

Comment:
Out of curiosity are those thresholds 17 & 15 for a specific reason or just to have something to put in there? (I had to check the db function to see what they were is why I'm wondering)

Reply:
the movement threshold could be anything, the time related thresholds I set for the purposes of convenience and enabling a short test period

Reply:
That makes sense, thanks! :)

Reply:
I just wasn't sure if those numbers were important haha